libs/openFrameworks/graphics/ofImage.cpp

```diff
@@ -209,23 +214,35 @@ static bool loadImage(ofPixels_<PixelType> & pix, const of::filesystem::path& _f
 		return ofLoadImage(pix, ofLoadURL(_fileName.string()).data);
 	}
 
-	auto fileName = ofToDataPath(_fileName, true);
+	auto fileName = ofToDataPathFS(_fileName, true);
 	bool bLoaded = false;
 	FIBITMAP * bmp = nullptr;
 
 	FREE_IMAGE_FORMAT fif = FIF_UNKNOWN;
+#ifdef OF_TARGET_WINDOWS
+	fif = FreeImage_GetFileTypeU(fileName.wstring().c_str(), 0);
```

Comment:
`std::filesystem::path::c_str()` will produce a native-appropriate `char *` so it's not necessary to pre-transform with wstring().